src/vis/MeasurementMap.tsx

```diff
@@ -126,6 +128,12 @@ const MeasurementMap = ({
       setSLayer(L.layerGroup().addTo(_map));
       setMLayer(L.layerGroup().addTo(_map));
       setLLayer(L.layerGroup().addTo(_map));
+
+      const search = new (GeoSearchControl as any)({
+        provider: new OpenStreetMapProvider(),
+        style: 'bar', // optional: bar|button  - default button
```

Comment:
The comment came from the library's documentation right? We can remove them here. Readers of this code can always look it up.